chips/nrf52/src/chip.rs

```diff
@@ -150,3 +150,9 @@ impl<'a, I: InterruptService + 'a> kernel::platform::chip::Chip for NRF52<'a, I>
         CortexM4F::print_cortexm_state(write);
     }
 }
+
+impl<'a, I: InterruptService + 'a> kernel::platform::chip::ChipThreadId for NRF52<'a, I> {
```

Comment:
This would have to be an `unsafe trait`, because we rely on implementations of this trait to be correct.